src/gitops/components/appset/YAMLTab.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
```

Comment:
Can we move this to a shared component or a utils component?  The Applications YAML tab will use this too